src/table/src/metadata.rs

```diff
@@ -219,6 +220,21 @@ impl TableMeta {
                         new_options.ttl = Some(*new_ttl);
                     }
                 }
+                ChangeOption::Twsc(key, value) => {
+                    if !value.is_empty() {
```

Comment:
When is the value empty? For implementing the `UNSET` later?

Reply:
same as ttl change the value could be empty like `ALTER TABLE ato SET 'compaction.twcs.time_window'='';` this will set the default value for that option